.travis.yml

```diff
@@ -1,9 +1,16 @@
 language: node_js
+sudo: false
 
 node_js:
- - iojs
- - 0.10
- - 0.12
+  - "0.12"
```

Comment:
probably add `6`, and go newest to oldest (makes the first / primary build in the matrix the latest version)